### PR TITLE
remove postcode lookup

### PR DESCRIPTION
**Pull request by cjrace**
Have removed the postcode lookup as added in #114, as that was only half the file, the full file is too big. Instead we'll just point teams to the ONS website to download from directly.

To compensate as far as the screening checks are concerned, I've pulled in the latest constituencies as a separate file until ONS updates the main Ward > PCon > LAD > LA lookup that we use elsewhere.

### CODE REVIEW

**Comment by cjrace** [comment]
Adding @rmbielby to review as this is effectively undoing what we did in #114

**Comment by cjrace** [comment]
[Remove PCon postcode lookup from screener](https://trello.com/c/RcsSrKHL/1386-remove-pcon-postcode-lookup-from-screener)

File: R/knownVariables.r [hunk 1]
```diff
@@ -85,9 +85,9 @@ edas <- suppressMessages(read_csv("data/english_devolved_areas.csv"))
 
 # PCons, this is quick and dirty, should take the chance to clean up when we move to using Pins
 pcons <- suppressMessages(read_csv("data/pcons.csv"))
-pcons_2024 <- suppressMessages(read_csv("data/postcode_to_pcon_2024.csv")) %>%
-  distinct(pconcd, pconnm) %>%
-  rename("pcon_code" = pconcd, "pcon_name" = pconnm)
+pcons_2024 <- suppressMessages(read_csv("data/pcon_2024_v2.csv")) %>%
+  rename("pcon_code" = PCON24CD, "pcon_name" = PCON24NM) %>%
```
Comment (rmbielby) [inline R/knownVariables.r]: You could just do this as one single line:
`select(pcon_code = PCON24CD, pcon_name = PCON24NM)`

File: data/pcon_2024_v2.csv
Comment (rmbielby) [inline data/pcon_2024_v2.csv:1]: Any particular reason for the "_v2" suffix on this filename?

File: data/pcon_2024_v2.csv [hunk 2]
```diff
@@ -0,0 +1,651 @@
+﻿PCON24CD,PCON24NM,PCON24NMW,ObjectId
```
Comment (rmbielby) [inline data/pcon_2024_v2.csv:1]: Is there any reason to keep the ObjectId field?

**Review by rmbielby — COMMENTED** [review]
Few thoughts from looking through the files:
- Are we going to just merge pcons_2024_v2.csv into pcons.csv at some point and apply a date of termination / date of introduction so we're consistent in how we deal with geographies and have only one base pcon file?
- If we're keeping both for now, do we want to rename the pcons.csv file to have a suffix like pre-2024?

File: data/pcon_2024_v2.csv
Comment (cjrace) [reply data/pcon_2024_v2.csv:1]: It's what ONS had called it

File: data/pcon_2024_v2.csv (hunk 2 above)
Comment (cjrace) [reply data/pcon_2024_v2.csv:1]: Just to keep faithful to the original file from ONS, so we've not transformed it in anyway (and to be lazy...)

File: R/knownVariables.r (hunk 1 above)
Comment (cjrace) [reply R/knownVariables.r]: Nifty! Have updated to this in my PR response commit

**Review by rmbielby — APPROVED** [review]

**Pull request merged**